lib/src/components/NxTreeView/NxTreeView.scss

```diff
@@ -18,16 +18,18 @@ $twisty-proportion-difference: 2px;
 */
 .nx-tree-view {
   box-sizing: border-box;
-  margin: var(--nx-spacing-4x) 0;
+  margin: var(--nx-spacing-4x) -8px;
 
   // Design specifies 300px wide not including the caret and its padding (which are themselves 30px)
   max-width: 330px;
```

Comment:
Does this need adjusting to account for the extra space on the left?

Reply:
> Does this need adjusting to account for the extra space on the left?

Direction from @dturner32 is a max-width of 360px including the focus states. https://github.com/sonatype/sonatype-react-shared-components/pull/408/commits/09e528f7a9a6fdd52c0b3fe7b1b1dda4662f2a86 and where I realized I shouldn't have included the negative margin: https://github.com/sonatype/sonatype-react-shared-components/pull/408/commits/1fda0c78374ce8ed751afe67c48f0e309dd02750

Reply:
> Direction from @dturner32 is a max-width of 360px including the focus states. 09e528f and where I realized I shouldn't have included the negative margin: 1fda0c7

If I'm not mistaken, the intent is effectively that the `.nx-tree-view__text` be 300px wide.  It is currently coming out to 323.75px wide.

Reply:
> If I'm not mistaken, the intent is effectively that the .nx-tree-view__text be 300px wide. It is currently coming out to 323.75px wide.

Oh nvm I think it's the `.nx-tree-view__child` content box that's supposed to be 300px wide. Which it is.

Reply:
> Oh nvm I think it's the .nx-tree-view__child content box that's supposed to be 300px wide. Which it is.

Something's still wrong though.  The mockups show the children (when clickable) as being 350px wide but in the impl they are currently 360px wide.
![image](https://user-images.githubusercontent.com/3630091/121035762-2ffe5000-c77c-11eb-81bd-051fb8257292.png)

Reply:
> > Oh nvm I think it's the .nx-tree-view__child content box that's supposed to be 300px wide. Which it is.
> 
> Something's still wrong though. The mockups show the children (when clickable) as being 350px wide but in the impl they are currently 360px wide.

That looks like @dturner32 either hasn't updated the mockups or that he's just showing an example in a constrained container.

Reply:
> > > Oh nvm I think it's the .nx-tree-view__child content box that's supposed to be 300px wide. Which it is.
> > 
> > 
> > Something's still wrong though. The mockups show the children (when clickable) as being 350px wide but in the impl they are currently 360px wide.
> 
> That looks like @dturner32 either hasn't updated the mockups or that he's just showing an example in a constrained container.
> 
> Seems like it's both. I have no problem with the mockups showing constrained examples but we should be stating the max width somewhere.

the mocks have just been updated to reflect this